components/VC/ExistingMosipVCItem/ExistingMosipVCItem.tsx

```diff
@@ -115,6 +122,15 @@ export const ExistingMosipVCItem: React.FC<
         onDismiss={DISMISS}
         translationPath={'VcDetails'}
       />
+
+      <MessageOverlay
```

Comment:
if we have this overlay in vc item components, will there be a chance that multiple overlays are shown at a time if multiple VC's are tampered at a time